charts/spark-operator-chart/tests/poddisruptionbudget_test.yaml

```diff
@@ -17,21 +17,22 @@ tests:
 
   - it: Should render spark operator podDisruptionBudget if podDisruptionBudget.enable is true
     set:
+      replicaCount: 2
```

Comment:
PDB tests were failing on the master branch so these are fixes to get them to pass.